### ISSUE TITLE
teams are not reopened when the app starts

### ISSUE THREAD

**Issue by cromega** [raelgc/scudcloud#482]
## ScudCloud Version

```
ScudCloud 1.31
Python 3.4.3
Qt 4.8.6
PyQt 4.10.4
SIP 4.15.5
```
## Distro and Desktop info
- Distribution: Mint
- Version/Release: 17.3
- Desktop environment: cinnamon
## Expected behavior

I open ScudCloud and I see all the teams I was signed in to
## Actual behavior

Most of the time I only see one team without the team sidebar being present. When I click on "Sign in to another team" and perform a sign-in the sidebar comes back with all the teams I expected to see when the app started.
## Steps to reproduce
1. Start ScudCloud .

**Comment by raelgc** [comment]
Hi @cromega, thanks for reporting that.

This is duplicate of #435.

Workaround: if you have the config file (`~/.config/scudcloud/scudcloud.cfg`), check if you have a row with `Domain` value. If not add it with your Slack domains (below are samples):

`Domain=https://scudcloud.slack.com/, https://example.slack.com/, https://foobar.slack.com/`

**Comment by cromega** [comment]
@raelgc ah, that works.

**Comment by cromega** [comment]
@raelgc: actually, I noticed that it kind of doesn't work lately. I have this issue again, almost every time I launch scudcloud. I'm running latest version and my config file contains all the domains.

**Comment by raelgc** [comment]
Wondering if this is related to the team names or something like that. Really weird. 

If you delete the config and start again, still able to face this issue?

**Comment by maiatoday** [comment]
I deleted my config file and added the domains again, still no teams :(
ubuntu 16.04
ScudCloud 1.37
Python 3.5.2
Qt 4.8.7
PyQt 4.11.4
SIP 4.17

**Comment by loonies** [comment]
Confirmed, no team switcher. `CTRL + 1,2,3,...` doesn't work either

**Comment by raelgc** [comment]
@maiatoday which desktop environment are you using? Can you, please, close ScudCloud and add the domains manually to the config file?

@loonies the switcher will work only when the teams are properly loaded.

**Comment by loonies** [comment]
What do you mean by "properly loaded"?

It worked fine yesterday. I did try to delete cache, config, changed settings, made sure that all teams are listed, restarted scudcloud, etc, etc. Still doesn't work.

If I go to "Sign in to another team..." I'm prompted to enter domain and after that a team is opened without need to log in as I'm already logged, but there is no sidebar team switcher.

/edit

I can click on "You’re already signed in to these teams" as well and a team is opened, but no team switcher.

**Comment by raelgc** [comment]
@loonies Thank you for adding more details. I just closed the app and re-open it, and I can see the issue. Please, open a new issue for this. I'll work in a fix.

**Comment by raelgc** [comment]
To be clear: no code was changed. Something changed in Slack side.

**Comment by maiatoday** [comment]
I am running unity 7.4.0 (don't judge me :smile_cat: ) 
After I deleted the config file no matter what I do in the app, it always has `Domain=@Invalid()` before it had a list of domains in there. I added my domains in there as per the description above and now it opens the first team but still no side bar. Let me know if you need more info

**Comment by raelgc** [comment]
@maiatoday I use Unity too. So, hope others don't judge us :)

This is a new issue, fixed few minutes ago: #518. 

Please, update and upgrade your Ubuntu. @masterkorp updated Arch (AUR) too.
